src/Captioning/File.php

```diff
@@ -143,6 +143,8 @@ public  function loadFromFile($_filename = null)
     {
         if ($_filename === null) {
             $_filename = $this->filename;
+        } else {
+            $this->filename = $_filename;
```

Comment:
Because on line 154 we are using `if (!($content = file_get_contents($this->filename))) {`